apps/desktop/src/lib/utils/selection.ts

```diff
@@ -8,12 +8,15 @@ import type { AnyFile } from '$lib/vbranches/types';
 
 export function getNextFile(files: AnyFile[], currentId: string): AnyFile | undefined {
 	const fileIndex = files.findIndex((f) => f.id === currentId);
-	return fileIndex !== -1 && fileIndex + 1 < files.length ? files[fileIndex + 1] : undefined;
+	const nextFile =
+		fileIndex !== -1 && fileIndex + 1 < files.length ? files[fileIndex + 1] : undefined;
+	return nextFile;
```

Comment:
It looks like this is a no-op change?

Reply:
I updated it a bit to this:
```ts
fileIndex > 0 && fileIndex + 1 < files.length ? files[fileIndex + 1] : undefined;
```
I'm checking if fileIndex is greater than 0 and within the bounds of the files array. If both conditions are true, the code returns the next file in the array. Otherwise, it returns undefined.

Can you suggest a better way here?

Reply:
The only change you're making here is declaring the variable `nextFile` and then immediately removing it. Perhaps you meant to modify the expression?
<img width="1038" alt="image" src="https://github.com/user-attachments/assets/6d9cadbf-9b2e-46b4-8e29-cac82081503d">

Reply:
Ah, I see. Sure, I'll fixed that